Makefile

```diff
@@ -63,6 +63,9 @@ E2E_TARGET ?= ./test/e2e/...
 
 E2E_KIND_VERSION ?= kindest/node:v1.24.7
```

Comment:
What happens if we specify this env var?
IIUC, this env var can not affect test env.

Reply:
This still has an effect on the `test-e2e`  target,  `1.24.7` is extracted from it and `run-test-e2e-1.24.7` will run

Reply:
Can we do `$(MAKE) E2E_KIND_VERSION=$(VERSION) run-test-e2e`, instead of different targets?

Reply:
> Can we do $(MAKE) E2E_KIND_VERSION=$(VERSION) run-test-e2e, instead of different targets?

It seems more intuitive.